src/app/components/ManualEntry.svelte

```diff
@@ -84,50 +132,87 @@
     font: 1em var(--main-font);
   }
 
-  input:focus {
+  table input:focus {
     background-color: var(--lite-white);
   }
+
+  table p {
+    display: inline;
```

Comment:
I strongly advice against setting `p` to have `display: inline`. `<p>` is semantically a block-level element, so the HTML spec actually treats it as a "block" even if it appears inline.